lib/v2/screens/dashboard/components/transactions_list/interactor/viewmodels/member_events.dart

```diff
@@ -0,0 +1,11 @@
+import 'package:equatable/equatable.dart';
+
+abstract class MemberEvent extends Equatable {}
+
+class LoadMemberDataEvent extends MemberEvent {
+  final String account;
+  LoadMemberDataEvent(this.account);
+
+  @override
+  List<Object> get props => [account];
+}
```

Comment:
Ok 

I guess the "on... " prefix makes it clear it's an event, otherwise, I would not agree that we "know" it's an event ;) 

But will go with the On...